### ISSUE TITLE
v1 default: Include Safe: false by default

### ISSUE THREAD

**Issue by marcandre** [rubocop-hq/rubocop#8156]
@bbatsov wrote in #5980 about `--safe-auto-correct`
> Let's implement the metadata and the options firsts, changing the defaults is easy afterwards. 😄

 Now would be a good time to ask ourselves if the current default is the right one. safe or unsafe by default?

`rubocop -a` does all autocorrections, including unsafe ones. One has to add `--safe-auto-correct` to exclude unsafe ones.

Should this be the default, or should it be the reverse?

**Comment by bbatsov** [comment]
That's a fair question. I guess by now most unsafe cops (at least those that are really problematic) have been marked as unsafe. I'm afraid that if we make unsafe the default, we'll have to make every existing cop without a default safe anyways, otherwise this would be a huge changes of the current behaviour (unless someone wants to carefully audit each cop, that is).

**Comment by marcandre** [comment]
> I'm afraid that if we make unsafe the default

Are you talking about the default for Cops? Or for the CLI? The Cops are Safe by default, but the CLI is currently unsafe by default. I assumed that assuming Cops to be safe by default was ok, my question is about if the CLI `rubocop -a` should include or exclude unsafe cops by default...

**Comment by bbatsov** [comment]
Ops, I got confused then. I thought we were discussing the cops in general. For the CLI probably safe auto-correct is a better default, although I'm wary of adding something named `unsafe-auto-correct` as it would probably scare people too much. :-)

**Comment by marcandre** [comment]
How about `--exclude-unsafe` and `--include-unsafe`, and `-a` is `--exclude-unsafe` by default and `-A` is `--include-unsafe` by default?

**Comment by bbatsov** [comment]
Sounds reasonable to me.

**Comment by paul** [comment]
This broke the autofix for `Style/FrozenStringLiteralComment`. I've tried hunting around, and I haven't been able to figure out why this one is "unsafe".

Also, this change broke a few other tools like Ale https://github.com/dense-analysis/ale/pull/3237, I'm unable to make this work until the fix their default. Is there something I can do in the meantime in the `.rubocop.yml` to either allow it to run/autofix "unsafe" cops, or mark this particular cop as "safe" so it can be run?

**Comment by marcandre** [comment]
Sorry for the troubles you are having @paul. Thanks for reaching out.

You can override the settings *per cop* with `Safe: true` and `SafeAutoCorrect: true`, in your `.rubocop.yml`:

```yaml
Style/FrozenStringLiteralComment:
  Safe: true
  SafeAutoCorrect: true
```

There are a few open issues that I hope will make it more obvious what is going on. We are planning on documenting why a cop isn't safe (e.g. `Style/FrozenStringLiteralComment` is that `str = 'silly'; str << ' example'` would fail...). We also want to show how many "unsafe" auto-corrections could be additionally be applied.
